pkg/service/rtcservice.go

```diff
@@ -166,6 +166,12 @@ func (s *RTCService) validate(r *http.Request) (livekit.RoomName, routing.Partic
 		}
 	}
 
+	createRequest := &livekit.CreateRoomRequest{
+		Name:       string(roomName),
+		RoomPreset: claims.RoomPreset,
+	}
+	SetRoomConfiguration(createRequest, claims.GetRoomConfiguration())
```

Comment:
this duplciates work that happens during `applyNamedRoomConfiguration` in room_lifecycle. we probably don't need to copy of the default room config into every signal session.

Reply:
I don't believe it does. `applyNamedRoomConfiguration` is copying it from named presets.. while this is copying it from `ClaimGrant`